src/handlers/poster-hubble/components/OverlayHubble.tsx

```diff
@@ -0,0 +1,27 @@
+import { OverlayProps } from '../../poster-base';
+import HubbleProgressBar from './HubbleProgressBar';
+import HubbleOrderView from './HubbleOrderView';
+
+export default function OverlayHubble({
+  // poster,
+  // posterTitle,
+  seconds,
+  posterIndex,
+  nextPoster,
+  pausePoster,
+}: OverlayProps) {
+  return (
+    <>
+      <HubbleProgressBar
+        seconds={seconds}
+        posterIndex={posterIndex}
+        hideClock={false}
+        // color={poster.color}
```

Comment:
?

Reply:
Loose ends that need to be tied when the core is updated. But I'd rather keep this in here and merge, to prevent having to keep rebasing everytime

Reply:
Then we can update the core with the color update, and then tie these loose ends up